src/LogoutButton.test.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import {shallow, mount} from "enzyme";
+import LogoutButton from "./LogoutButton";
+
+it("renders LogoutButton without crashing", () => {
+  mount(<LogoutButton />);
+});
+
+it("renders correctly (snapshot testing)", () => {
```

Comment:
When we have snapshot testing, is the 'renders without crashing' needed? Please read about it to see what the best practice is.